src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;

```

Comment:
move font to html selector and margin into body
you can apply to * only box-sizing